src/cascadia/LocalTests_TerminalApp/TerminalApp.LocalTests.AppxManifest.prototype.xml

```diff
@@ -28,7 +28,7 @@
   </Properties>
 
   <Dependencies>
-    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17763.0" MaxVersionTested="10.0.18362.0" />
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.18362.0" MaxVersionTested="10.0.19041.0" />

```

Comment:
We may want to keep this one at 17763 -- I know that sounds crazy, but the only limiting factor that kept us on MIN 18362 was _XAML Islands_. These tests don't use xaml islands :smile:

Reply:
(so if these will run on older versions of the _build agent_, that might make things easier on us)